### ISSUE TITLE
Resolve Incorrect heading levels on Toast

### ISSUE THREAD

**Issue by georgehager** [CDCgov/prime-simplereport#4564]
## Background

We've added axe accessibility testing to our cypress e2e tests, and have some failures that need to be resolved. Currently, our Toast notification triggers a11y errors across multiple pages, as shown by our e2e tests.

## Action Requested

- Remove the `rules` in the `checkA11y()` call that allow the axe cypress tests to ignore the Toast errors
- Fix the a11y issues on the Toast. To quote @emyl3, a possible solution may look like:
> I was checking out https://dequeuniversity.com/library/aria/alert example of an alert — they don’t seem to be adding any heading tags… Since we are passing our own `<Alert>` component to those toast notifications I don’t see any issue with changing the `<h3>` to a styled `<div>/<span>` so the heading order makes sense

## Acceptance Criteria

Running a Deque audit and the e2e tests result in no a11y errors related to the Toast.

## Resources & Notes

- The Toast errors can be easily found in e2e specs 02, 03, 08 with a test and comments similar to this:
```
// failing a11y test
    // error applies to the toast
    // observe this by adding cy.wait(5000); to wait for the toasts to disappear
    cy.checkA11y(
        {
          exclude: [],
        },
        {
          rules: {
            'heading-order': { enabled: false },
          },
        },
    );
```
- For the affected calls to `checka11y()` there may be multiple errors, not all of which are related to the Toast. It's not necessary to fix those at this time.
- Screenshots
![toast-error-1](https://user-images.githubusercontent.com/89797785/197779509-f43f1685-f84d-47e6-b2b0-39ae05985dc3.png)
![toast-error-2](https://user-images.githubusercontent.com/89797785/197779511-8a6db21b-5c02-4401-8ac0-dc6585f4af43.png)
![toast-error-3](https://user-images.githubusercontent.com/89797785/197779530-8418ce2c-d7fc-4915-aa4b-04d8674e9f57.png)
